config/audit_log/user_action_events.yml

```diff
@@ -1,4 +1,60 @@
----
 user_login:
   event_type: authentication
   details: User logged in
+
+sign_in: 
+  event_type: sign_in
```

Comment:
event_type is defined twice under `sign_in` the duplicate keys could lead to errors in parsing so I think nesting under a list would help avoid that ambiguity.

```
sign_in: 
  - event_type: sign_in
    details: Sign in {{ on VA.gov }}
  - event_type: verify_identity
    details: Verify identity {{ on ID.me }}
```